apps/docs/pages/docs/api-docs.mdx

```diff
@@ -242,10 +242,17 @@ For the `edit` property, it can take the following:
 | -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | validate       | a function that takes the field value as a parameter, and that returns a boolean                                                                                             |
 | format         | a string defining an OpenAPI field format, overriding the one set in the generator. An extra `file` format can be used to be able to have a file input                       |
+| input          | a React Element that should receive [CustomInputProps](#custominputprops). For App Router, this element must be a client component.                                          |
 | handler        | an object that can take the following properties                                                                                                                             |
 | handler.get    | a function that takes the field value as a parameter and returns a transformed value displayed in the form                                                                   |
 | handler.upload | an async function that is used only for formats `file` and `data-url`. It takes a buffer as parameter and must return a string. Useful to upload a file to a remote provider |
 
+<Callout type="info" emoji="ℹ️">
+
+Custom input will only work with `format: "string"`.
```

Comment:
Why ?

Reply:
This seems weird to force the user to add a custom option to allow overriding inputs, should we force the format if user provide a custom input ?

Reply:
Yes we can do that